mobile/native/src/db/models.rs

```diff
@@ -1557,6 +1573,17 @@ pub mod test {
         };
 
         assert_eq!(expected, loaded);
+
+        // Verify that we can delete the right spendable output based on its outpoint
+        SpendableOutputQueryable::delete(outpoint, &mut connection).unwrap();
+
+        match SpendableOutputQueryable::get(outpoint, &mut connection) {
+            // Could not find spendable output as expected
+            Err(Error::NotFound) => {}
+            _ => {
+                panic!("Expected to not be able to find deleted spendable output");
```

Comment:
❓ why do we need to panic here?

Reply:
I just followed the pattern of the other test where we delete a row. The point is that we are expecting a specific error when loading a deleted row, so we can fail the test in all other cases.